examples/feature_monorepo/README.md

```
@@ -0,0 +1,13 @@
+# /pub/examples/partcad/feature_monorepo
+
+Demonstrates how to create a monorepo with multiple packages in it
+
+## Sub-Packages
+
+### [package-b](./package-b/README.md)
+
+### [package-a](./package-a/README.md)
+
+<br/><br/>
```

Markdownlint (0.37.0)

Element: br
Inline HTML

(MD033, no-inline-html)